### DISCUSSION TITLE
Warning message when compiling NanoEls H2

### DISCUSSION THREAD

**Discussion by EdFleta** [kachurovskiy/nanoels#90] [Q&A]
Hi, since my first implementation of NanoEls H2, Arduino IDE returns the following warnings: 
"NanoEls-V4-CL.ino:72:0: warning: "B0" redefined #define B0 4" 

the same warning also occurs for B0 0, B1 5 and B1 1.

It does not prevent the actual loading of the compiled software but I just wonder whether it could have an impact somewhere down the line.

any idea ? 
eddie

**Comment by kachurovskiy** [comment]
Hi @EdFleta, I would suggest copying https://github.com/kachurovskiy/nanoels/blob/main/h2/h2.ino and re-doing your changes on top of it, looks like there's been some copy-pasting mistake while adopting the code to your particular case.

Alternatively, you can post it on https://gist.github.com/ and share the link here for us to take a look what's going on. Cheers!

**Reply by EdFleta** [reply]
Hi Maxim,
The last "clean up" version produces no warning at all on my Arduino IDE and compiles correctly. I guess the issue can be closed as far as I am concerned.
best regards,

eddie
